docs/source/requirements.rst

```diff
@@ -2,8 +2,8 @@ Requirements
 ============
 Before installing Quipucords in your environment, review the following guidelines about installing and running Quipucords:
 
-- You must install Quipucords on a system that is running Red Hat Enterprise Linux 6 or 7.
+- You must install Quipucords on a system that is running Red Hat Enterprise Linux 6 or 7, CentOS 7, or Fedora 26 or 27. For an installation on Fedora, the scripted installation of Quipucords might not work correctly on all systems, so you might have to use the step-by-step installation instructions.
 - The system that Quipucords is installed on must have access to the systems to be discovered and inspected.
```

Comment:
"might  ... might" repeated. Maybe mix it up? "The scripted installation of Quipucords is only fully supported on RHEL and CentOS, Fedora users are advised to use the step-by-step installation instructions". I don't know, maybe that "supported" word is too radioactive.

Reply:
Yes, I would like to stay far, far away from the "fully supported" phrase in that context without being an official RH product with a defined support process. The IBM Style Guide, which RH uses, also strongly censures the use of "recommend" (kind-of like your suggested "advised" above) for similar legal reasons. I will take another look at the paragraph and see if I can work in "best practice" (not as loaded a term) somewhere to describe the current recommendation. (And just an FYI, chaining words and repeating words (might/might) is a fairly standard practice in tech writing...providing users (esp. non-native English speakers) and translators with predictable word patterns.)

Reply:
The statement now says this:

For an installation on Fedora, the scripted installation of Quipucords might not work correctly on all systems. Therefore, the best practice for a Fedora installation is to use the step-by-step installation instructions.